lib/utils/getClientPath.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+/* eslint-disable
+  no-unused-vars
+*/
+
+function getClientPath(options) {
+  // this is where we can change the source of the client implementation
+  // eg. WebsocketClient or a user-provided implementation
+  return require.resolve('./../clients/SockJSClient');
+}
+
+module.exports = getClientPath;
```

Comment:
It should be static method for SockJSClient, i.e.
BaseClient should have `static getClientPath(options) { throw new Error('Need implementation'); }` and
SockJSClient should have `static getClientPath(options) { return require.resolve('./../clients/SockJSClient'); }`
For other clients too